### ISSUE TITLE
Fix: include returns undefined for classes

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/scripts#1075]
For
```Javascript
class test { myMethod() {}}
```
it gives "undefined".
For
```Javascript
class test { myMethod() {}}
test
```
it gives "class test { myMethod() {}}".

This is caused by a bugfix in graaljs.

Affected:
- Verbs
- Plugins
- Settings

Possible solutions:
- `(class C {})`
- `class C {}; C`
- use old graaljs version (19.0.0)

see https://github.com/PhoenicisOrg/phoenicis/pull/2040

**Comment by madoar** [comment]
I'm in favor of the third solution:

> use old graaljs version (19.0.0)

because it is the quickest solution, leads to no problems for us as far as I know, and is also the easiest solution to reverse when we found a solution for https://github.com/PhoenicisOrg/phoenicis/issues/2042

**Comment by plata** [comment]
I would vote for one of the other solutions because
- it allows us to update graaljs
- I have the feeling that we will not be able to use ES modules or node modules

**Comment by madoar** [comment]
What about extending our own include mechanism to provide explicit exports?
We could even try to execute included scripts in an isolated environment (i.e. their own polyglot context) and inject the populated result object in the calling script.

**Comment by plata** [comment]
What do we gain with explicit exports?
I don't think an isolated environment would work e.g. for engine plugins.

**Comment by madoar** [comment]
Explicit exports make the code easier to understand. In addition we are able to export multiple things in one script. 

About executing the included scripts in an isolated environment:
It seems like this is blocked by https://github.com/oracle/graal/issues/631

**Comment by madoar** [comment]
@plata I think this is fixed by #1076. Can we close this?
